core/src/main/scala/org/apache/spark/internal/config/Status.scala

```diff
@@ -56,7 +56,7 @@ private[spark] object Status {
     .createWithDefault(Int.MaxValue)
 
   val APP_STATUS_METRICS_ENABLED =
```

Comment:
While we are consolidating this, shall we update `APP_STATUS_METRICS_ENABLED` -> `METRICS_APP_STATUS_SOURCE_ENABLED` together?